salt/daemons/__init__.py

```diff
@@ -1,24 +1,16 @@
-# -*- coding: utf-8 -*-
 """
 The daemons package is used to store implementations of the Salt Master and
 Minion enabling different transports.
 """
-from __future__ import absolute_import, print_function, unicode_literals
 
 import logging
-
-# Import Python Libs
 import sys
 from collections.abc import Iterable, Mapping, Sequence
 
-# Import Salt Libs
 from salt.ext import six
 
 log = logging.getLogger(__name__)
 
-if sys.version_info[0] == 3:
-    six.string_types = (six.text_type, six.binary_type)
```

Comment:
This is the only thing I wanted to change by this PR. `six.string_types` is set in `salt.ext.six` and only there. It must not be updated outside the `six` module. Moreover this like updates the `six.string_types` value for the loaded module so it changes the `string_types` for all the code in the current process.

One more interesting thing is: runtests is vulnerable for this while pytest is not.

Reply:
Heh, usually it's the other way around.....

Reply:
I uh... why are we setting `six.string_types` in *this* module? 😬 

I mean, we're not now but... yikes!

Reply:
I spent several hours second-guessing my own existence over this. I thought I was going crazy!